frontend/app/tests/e2e/pages/assets-manager-page.ts

```diff
@@ -28,7 +28,6 @@ export class AssetsManagerPage {
 
   addIgnoredAsset(asset: string) {
     this.searchAsset(asset);
-    cy.wait(1000);
     cy.get(
```

Comment:
add timeout here instead (?)

Reply:
I need to check but I am guessing we have to verify that the drop-down is visible before continuing.